nservicebus/rabbitmq/configuration-api.md

```diff
@@ -78,4 +78,25 @@ In order for the transport to use you new connection manager you need to registe
 ### Controlling behavior when broker connection is lost
 By the default the RabbitMQ transport will trigger the on critical error action when it continuously fails to connect to the the broker for 2 minutes. This can now be customized using the following configuration setting: (values must be parsable to `System.TimeSpan`)
 
-<!-- import rabbitmq-custom-breaker-settings -->
\ No newline at end of file
+<!-- import rabbitmq-custom-breaker-settings -->
+
+
+### Changing routing topology
+By default the RabbitMQ transport will creates separate exchanges for each message type, including inherited types, being published in the system. This means that support polymorphic routing as well as multiple inheritance for events is supported since each subscriber will bind their input queue to the relevant exchange based on the event type that is has handlers for.

```

Comment:
* RabbitMQ transport will **creates** separate --> create
* This means that **support** polymorphic routing as well as multiple inheritance for events --> support is not required, does not read well
* since each subscriber will bind **their** input --> shouldn't it be "its"
* type that **is** has handlers for. --> it